core/scripts/sql/update/user.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `texera_db`.`user` 
+ADD COLUMN `permission` INT NOT NULL DEFAULT 0 AFTER `google_id`,
+CHANGE COLUMN `name` `email` VARCHAR(256) NOT NULL;
+
+ALTER TABLE `texera_db`.`user` 
+ADD COLUMN `name` VARCHAR(256) NULL DEFAULT NULL AFTER `uid`;
```

Comment:
Suppose I want to update a db for a deployment. After running this script, can I use dev mode (not google login) to login? I think the `name`->`email` change will break local login?

Reply:
I will test and fix it if there is any issue.